spk/plexpy-custom/src/config.ini

```diff
@@ -0,0 +1,3 @@
+[General]
+launch_browser = 0
+web_port = 8183
```

Comment:
Please choose a different port. See https://github.com/SynoCommunity/spksrc/wiki/SynoCommunity-Used-Ports#how-to-choose-a-port, the `xyz` sequence you chose doesn't follow our conventions.